site/reference/cli/fn/eval/README.md

```diff
@@ -103,8 +103,8 @@ fn-args:
   If using never, kpt will only use images from the local cache.
 
 --include-meta-resources, m:
-  If enabled, meta resources (i.e. `Kptfile` and `functionConfig`) are included
-  in the input to the function. By default it is disabled.
+  (DEPRECATED) include-meta-resources is no longer necessary because meta
+  resources are now included by default.
```

Comment:
Should we list what version they started being included by default? Here and below

Reply:
Good suggestion. included the version now. And also improved the `kpt pkg get` reference to include `--for-deployment`.

Thanks for the review.